message/messaging_service.hh

```diff
@@ -8,6 +8,7 @@
 
 #pragma once
 
+#include "db/config.hh"
```

Comment:
Please move to .cc, a rapidly-changing header should be avoided in other headers.